lib/src/chart/line_chart/line_chart_data.dart

```diff
@@ -1478,6 +1478,9 @@ class LineTouchTooltipData with EquatableMixin {
   /// Forces the tooltip to shift vertically inside the chart, if overflow happens.
   final bool fitInsideVertically;
 
+  /// Forces the tooltip container to top of the line, default 'false'
+  final bool tooltipBoxTopPositioned;
```

Comment:
it is not descriptive enough (I mean it doesn't tell that it will draw outside of the chart's box).
we can rename it to `showOnTopOfTheChartBoxArea`